udata_front/theme/gouvfr/datagouv-components/src/components/DataserviceLinks/DataserviceLinks.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+    <div>
+        <div 
+            class="border border-blue-400 fr-p-2w fr-dataservice__box"
+            v-if="(!showSwaggerBox && dataservice.endpoint_description_url) || dataservice.base_api_url"
+        >
+            <div class="fr-grid-row fr-grid-row--gutters fr-grid-row--middle">
+                <div class="fr-col-12 fr-col-sm">
+                    <h2 class="fr-h6 fr-mb-0 text-blue-400">{{ t("API documentation") }}</h2>
+                </div>
+                <div class="fr-col-auto flex">
+                <a
+                    class="fr-btn"
+                    :href="dataservice.endpoint_description_url"
+                    target="_blank"
+                    rel="ugc nofollow noopener"
+                    v-if="!showSwaggerBox && dataservice.endpoint_description_url"
+                >
+                    {{ t('Technical Documentation') }}
+                </a>
+                <a
+                    class="fr-btn"
+                    :class="showSwaggerBox ? '' : 'fr-ml-3v fr-btn--secondary'"
+                    :href="dataservice.base_api_url"
+                    target="_blank"
+                    rel="ugc nofollow noopener"
+                    v-if="dataservice.base_api_url"
+                >
+                    {{ t('Business documentation') }}
+                </a>
+                </div>
+            </div>
+        </div>
+        <br />
+        <div class="border border-blue-400 fr-p-2w fr-dataservice__box fr-dataservice__swagger" v-if="showSwaggerBox">
+            <header
+                class="fr-p-2v flex items-center justify-between relative"
```

Comment:
```suggestion
                class="fr-p-2v flex items-center justify-between"
```

The relative class seems unnecessary here